### DISCUSSION TITLE
Model Viewer Question

### DISCUSSION THREAD

**Discussion by emincanyuksel** [google/model-viewer#4232] [General]
hello, is it possible to play a video on an object in the model viewer, for example, on a screen, how can I do it?

**Comment by elalish** [comment]
See several options here: https://modelviewer.dev/examples/scenegraph/#animatedTexturesExample

**Reply by emincanyuksel** [reply]
This resource will definitely help me and I have another question for you. The .glb file on my site appears as src. Is there a way to hide or encrypt it?

**Reply by elalish** [reply]
Not well, but sort of, see the discussion in #1886.

**Reply by emincanyuksel** [reply]
I looked at the source but I didn't quite understand what to do, can you tell me what to do?

**Reply by emincanyuksel** [reply]
> [Pek iyi değil ama bir bakıma #1886'daki](https://github.com/google/model-viewer/discussions/1886) tartışmaya bakın .

can you help me ?

**Reply by elalish** [reply]
The short answer is you can't hide it, not really. You can do some tricks to make it a little harder to find, but it won't stop anyone who's serious: https://github.com/google/model-viewer/discussions/1886#discussioncomment-2585049

**Reply by emincanyuksel** [reply]
> The short answer is you can't hide it, not really. You can do some tricks to make it a little harder to find, but it won't stop anyone who's serious: [#1886 (comment)](https://github.com/google/model-viewer/discussions/1886#discussioncomment-2585049)

I have a question, I don't quite understand how to use the file here. Is there a sample file?
https://modelviewer.dev/examples/scenegraph/#animatedTexturesExample
